functions/submit-finding.js

```diff
@@ -99,15 +102,31 @@ exports.handler = async (event) => {
     const sessionToken = authorization.split("Bearer ")[1];
     const confirmed = await Moralis.Cloud.run("confirmUser", {
       sessionToken,
-      address,
-      username: handle,
+      username: user,
     });
     if (!confirmed) {
       return {
         statusCode: 401,
         body: "Unauthorized",
       };
     }
+    if (attributedTo !== user) {
+      const url = `${event.headers.origin}/.netlify/functions/getUser?id=${attributedTo}`;
```

Comment:
Does this work?

```suggestion
      const url = `/.netlify/functions/getUser?id=${attributedTo}`;
```

Reply:
Yes, it does. And in fact, node-fetch only allows using absolute urls. I can look into another work-around if this is not acceptable.